lib/service/servicecfg/windows.go

```diff
@@ -162,4 +172,9 @@ type LDAPConfig struct {
 	ServerName string
 	// CA is an optional CA cert to be used for verification if InsecureSkipVerify is set to false.
 	CA *x509.Certificate
+	// LocateServer automatically locates the LDAP server using DNS SRV records.
+	// https://ldap.com/dns-srv-records-for-ldap/
+	LocateServer
```

Comment:
I would move this up next to `Addr`.

Also, do we need to embed it, or can we give it a name?